stern.hcl

```diff
@@ -0,0 +1,22 @@
+description = "Multi pod and container log tailing for Kubernetes"
+binaries = ["stern"]
+test = "stern --version"
+
+platform "darwin" "amd64" {
```

Comment:
These URLS are all identical so you can just get rid of all these blocks and move source to the top level.

Reply:
Ah you are totally correct, I used `hermit manifest create`, perhaps I can add a heuristic in that?

> hermit manifest create https://github.com/stern/stern/releases/download/v1.32.0/stern_1.32.0_darwin_arm64.tar.gz > stern.hcl